### ISSUE TITLE
FYI - Compile Warnings on treepy.el (from treepy-0.1.1 ELPA, MELPA, etc.)

### ISSUE THREAD

**Issue by harmonicalchemy** [volrath/treepy.el#8]
Hi, I am not sure if you know about this, but I kind of worries me as I use **`treepy.el`** as dependent on **magit** and other packages...

I periodically restart my Emacs from scratch _(i.e., with empty elpa directory etc.)_ to make sure changes to my config still work with the base system refreshed and installed from scratch all over again...  I save the log to chase down any problems that may come back later... Not meaning to "chase" you or anyone else, God Forbid! LOL Only to chase the code (for my own purposes or to help others like you)

Anyway.. Lately I have noticed a few compile errors coming from **`treepy.el`**  Here are the details:

## Package Details:
```yaml
Package treepy is dependency.

     Status: Installed in ‘treepy-0.1.1/’ (unsigned).
    Version: 0.1.1
     Commit: b40e6b09eb9be45da67b8c9e4990a5a0d7a2a09d
    Summary: Generic tree traversal tools
   Requires: emacs-25.1
Required by: ghub-3.3.0
   Homepage: https://github.com/volrath/treepy.el
   Keywords: lisp maint tools 
 Maintainer: Daniel Barreto <daniel.barreto.n@gmail.com>
     Author: Daniel Barreto <daniel.barreto.n@gmail.com>
Other versions: 0.1.1 (MELPA-Stable), 20191108.2217 (MELPA-Unstable).
```

## Compiler Warnings from Log:
_(included short tail above warning to show context)_

```yaml
Compiling file /Users/alisha/.emacs.d/elpa/magit-popup-2.13.2/magit-popup.el at Mon Jun 29 17:17:12 2020
Entering directory ‘/Users/alisha/.emacs.d/elpa/magit-popup-2.13.2/’
 
Compiling internal form(s) at Mon Jun 29 17:17:13 2020
Leaving directory ‘/Users/alisha/.emacs.d/elpa/git-commit-2.90.1’
 
Compiling file /Users/alisha/.emacs.d/elpa/git-commit-2.90.1/git-commit.el at Mon Jun 29 17:17:13 2020
Entering directory ‘/Users/alisha/.emacs.d/elpa/git-commit-2.90.1/’

In treepy--with-loc:
treepy.el:159:35: Warning: ‘'node’ is a malformed function
treepy.el:159:35: Warning: ‘'context’ is a malformed function
treepy.el:162:51: Warning: ‘t’ called as a function
treepy.el:233:1: Error: Invalid function: 'node
```

**Comment by volrath** [comment]
@harmonicalchemy Thanks for the report! which Emacs version are you using?

**Comment by jackhill** [comment]
I'm having this error with Emacs 27.1-rc1. It works for me with Emacs 26.3.

**Comment by volrath** [comment]
Should be fixed now in `0.1.2` (#6).
